### PR TITLE
Adds 8 brand new "fun" AI lawsets

### PR DESCRIPTION
**Pull request by Hopekz**
Felt like making something so this PR adds 8 new original lawsets that should create some interesting RP scenarios.

None are on by default so if someone wants any one of them to be round-start that would probably be a separate PR otherwise these should be print-able unless I'm missing something.

:cl:  Hopek
rscadd: add: Added 8 original and "fun" non malicious lawsets to print. Partybot, TravelGuide and Friendbot, PlantFriend, GameMaster, FitnessCoach, Educator and Mediator
/:cl:

### CODE REVIEW

**Comment by Cartlord** [comment]
I actually really like the idea of lawsets that work best on 'borgs rather than A.I.s, though I feel like Chefbot wouldn't work well since (if memory serves) silicons don't actually have any way to harvest plants or cook food.

**Comment by drderp3635** [comment]
cheftbot and partybot both don't have lines that translate to "don't welder crew to death" 
as far as I can tell I don't see a reason why a chefbot borg couldn't kill a person to make food with, at least according to their laws.

**Comment by Hopekz** [comment]
> as far as I can tell I don't see a reason why a chefbot borg couldn't kill a person to make food with, at least according to their laws.

@Cartlord 
@drderp3635 

Check again I re-wrote them.
I forgot I was PR'ing to Yogstation so I have to hugbox them.

**Comment by Burger1998** [comment]
can borgs even make food other than spamming icecream?

**Comment by Hopekz** [comment]
> can borgs even make food other than spamming icecream?

I created it as an RP law so they'd be forced to work with people that can until someone codes in some way for them to do it themselves but thinking about it maybe it is too much to ask for so if people want I can remove it from the PR

**Comment by Moja10** [comment]
the chef one looks a little difficult to do unless you are an exact borg type. what are you supposed to if you were an Asimov AI turned chefbot AI? Engiborg? I think lawsets work best when they can transition well. Asimov AI can stop worrying about humans when changed to Reporter, and start worrying about interesting things and such. The party one works because the AI can do things to facilitate a party or a fun environment. The chef bot one basically only works if you are a service borg.

**Comment by Moja10** [comment]
otherwise I support more fun laws

**Comment by Hopekz** [comment]
@Moja10 

I went ahead and just removed the chef one since it seems to be brought up as an issue by multiple people + added a few more that I think will fit in pretty well.

**Comment by MajManatee** [comment]
i mean yeah sure? Im personally not particularly interested in these but more is good

**Comment by Hopekz** [comment]
> i mean yeah sure? Im personally not particularly interested 

If you don't care and have no feedback to offer besides saying you don't care then why bother commenting ?
Whenever you make a PR that I don't care about I don't comment that I don't care I just leave it alone

**Comment by MajManatee** [comment]
because i dont care

File: code/datums/ai_laws.dm [hunk 1]
```diff
@@ -187,6 +187,70 @@
...
+                    "You must provide accurate and helpful information about local customs, attractions, and safety precautions.",\
+                    "You must ensure humans have a positive and memorable travel experience, without conflicting with the First Law.",\
+                    "You must promote responsible and sustainable tourism practices among humans.")
+
+/datum/ai_laws/friendbot
+    name = "Friendbot"
+    id = "friendbot"
+    inherent = list("You must always be supportive and kind to humans, without causing harm.",\
+                    "You must help humans develop and maintain positive relationships with one another.",\
+                    "You must provide a listening ear and empathetic responses to humans in need, as long as it does not conflict with the First Law.",\
```
Comment (ReddicusDragon) [inline code/datums/ai_laws.dm:211]: empathy? In yogstation???

**Review by ReddicusDragon — APPROVED** [review]

**Comment by Hopekz** [comment]
> because i dont care

Alright now you just proved that you care because you felt compelled to respond twice and might even do it a third time

**Comment by Bibby0110** [comment]
> > because i dont care
> 
> Alright now you just proved that you care because you felt compelled to respond twice and might even do it a third time

youre weird hopek

**Pull request merged**